go/kbfs/libkbfs/kbfs_ops.go

```diff
@@ -1801,26 +1855,60 @@ func (fs *KBFSOpsStandard) initTlfsForEditHistories() {
 				"Handle %s for existing folder unexpectedly has no TLF ID",
 				h.GetCanonicalName())
 		}
+		time.Sleep(fs.config.Mode().BackgroundWorkPeriod())
+	}
+}
+
+func (fs *KBFSOpsStandard) startInitSync() (
+	context.Context, context.CancelFunc) {
+	fs.initLock.Lock()
+	defer fs.initLock.Unlock()
+	ctx := CtxWithRandomIDReplayable(
+		context.Background(), CtxFBOIDKey, CtxFBOOpID, fs.log)
+	ctx, cancel := context.WithCancel(ctx)
+	if fs.initSyncCancel != nil {
+		fs.initSyncCancel()
 	}
+	fs.initSyncCancel = cancel
+	return ctx, cancel
 }
 
 func (fs *KBFSOpsStandard) initSyncedTlfs() {
+	if fs.config.MDServer() == nil {
+		return
```

Comment:
Do we need to try init'ing again later or is this process just gonna die without ever getting a non-nil `mdserver`?

Reply:
We don't need to retry explicitly because we now call this function in the `Push...StatusChange()` functions, one of which will fire when the mdserver connectivity changes.  So as far as I can tell, it shouldn't ever get stuck because of this return.